rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue

```diff
@@ -491,17 +516,13 @@ export default defineComponent({
       this.totalTime = Date.now() - this.startTime
       this.populateLogsProm = undefined
     },
-    observeFiltered(lookupFunc: () => IObservableArray<ExecutionOutputEntry> | undefined) {
-      autorun((reaction) => {
-        const output = lookupFunc()
-        if (!output) return
-        reaction.dispose()
-        this.entries = output
-
-        if (this.mfollow) {
-          this.scrollToLine(this.entries.length)
-        }
-      })
+    flattenObj(obj = {}) {
+      return Object.keys(obj || {}).reduce((acc, cur) => {
+        if (typeof obj[cur] === 'object') {
+          acc = { ...acc, ...this.crushObj(obj[cur])}
```

Comment:
what is `crushObj` ?

Reply:
I missed the rename, it was supposed to be flattenObj https://github.com/rundeck/rundeck/pull/8554/commits/f3a3f09dd838c7cb5658fd46404448a4691d2034